CHANGELOG.md

```diff
@@ -1,13 +1,21 @@
-## [project-title] Changelog
+# FL Accelerator Changelog
 
-<a name="x.y.z"></a>
-# x.y.z (yyyy-mm-dd)
+##  September 2022 release
```

Comment:
Could we have sub sections like

### Repository Structure
- ...

### Provisioning
- ...

### Documentation
- ...

### FL Experience
(put metrics + factory here)